docusaurus/docs/get-started/get-started.mdx

```diff
@@ -180,7 +180,7 @@ App and data source plugins can have a backend component written in Go. Backend
 - Connect to non-HTTP services to which a browser normally can’t connect. For example, SQL database servers.
 - Keep state between users. For example, query caching for data sources.
 - Use custom authentication methods and/or authorization checks that aren’t supported in Grafana.
-- Use a custom data source request proxy. To learn more, refer to [Backend developer resources](https://grafana.com/docs/grafana/latest/developers/plugins/backend/#resources).
+- Use a custom data source request proxy. To learn more, refer to [Backend developer resources](./introduction/backend-plugins#resources).
```

Comment:
```suggestion
- Use a custom data source request proxy. To learn more, refer to [Backend developer resources](../introduction/backend-plugins#resources).
```

The website seems to figure this out, but I think should be `..`

Reply:
For some reason, your suggest produces a "Page Not Found" error, but the original link works. I'm not sure why, but @jackw may know.

Reply:
It's due to the `slug: /` in the metadata at the top of the file. This means where it lives on the file system (e.g. inside `get-started` directory) is irrelevant. Check the other internal links in this file and you'll see none of them try to go up a level out of the current directory. As such I don't think this suggestion is gonna work.